Cmdline/Action/Install.cs

```diff
@@ -129,7 +129,7 @@ public int RunCommand(CKAN.KSP ksp, object raw_options)
 
                 string message = String.Format("Too many mods provide {0}. Please pick from the following:\n", ex.requested);
 
-                int result = -1;
+                int result;
```

Comment:
How could it? Caught exceptions will return without using it. Uncaught will transfer the program flow out of the block and thus send result out of scope.